dvc/scm.py

```diff
@@ -123,3 +126,61 @@ def resolve_rev(scm: "Git", rev: str) -> str:
             if len(ref_infos) > 1:
                 raise RevError(f"ambiguous Git revision '{rev}'")
         raise RevError(str(exc))
+
+
+def get_revs_from_args(
+    scm: "Git",
+    head_revs: Optional[List[str]] = None,
+    num: int = 1,
+    all_branches: bool = False,
+    all_tags: bool = False,
+    all_commits: bool = False,
+) -> OrderedDict:
```

Comment:
Please don't use `OrderedDict` as `dict` is guaranteed to be ordered based on insertions from 3.7+ onwards.